### ISSUE TITLE
Segment + GA user signup event

### ISSUE THREAD

**Issue by jordanh** [ParabolInc/parabol#3974]
## Issue - Enhancement

TL;DR we want to reliably track new account creations in Google Analytics, and GA only gives you 3 fields. We need to set these fields

In order to get Segment `track` events into GA, we _must_ [use the following property schema](https://segment.com/docs/connections/destinations/catalog/google-analytics/):

```json
{
  "userId": "12345",
  "action": "track",
  "event": "Logged In",
  "properties": {
    "category": "Account",
    "label": "Premium",
    "value": 50
  }
}
```

We have a Segment event today called `Account Created` and set `isInvited` to `false` in order to detect "patient zero". For GA, we need this to be:

{
  "userId": "12345",
  "action": "track",
  "event": "Account Created",
  "properties": {
    // ... existing props ...
    "category": "All",
    "label": "isInvited",
    "value": (0 | 1) # 0 = not invited, 1 = invited
  }
}

### Acceptance Criteria (optional)
   - added addn'l props along the lines of the schema above
   - @jordanh told, so segment and GA configurations can be adjusted (esp. filter settings)

- **Estimated effort:** 2 hours ([see CONTRIBUTING.md](https://github.com/ParabolInc/parabol/blob/master/CONTRIBUTING.md#points-and-sizes))
